src/coreclr/jit/codegenloongarch64.cpp

```diff
@@ -9313,7 +9313,17 @@ void CodeGen::genFnPrologCalleeRegArgs()
                     // So, after we save `A7` on the stack in prolog, it has to copy the stack slot of the split struct
                     // which was passed by the caller. Here we load the stack slot to `REG_SCRATCH`, and save it
                     // on the stack following the `A7` in prolog.
-                    GetEmitter()->emitIns_R_R_Imm(INS_ld_d, size, REG_SCRATCH, REG_SPBASE, genTotalFrameSize());
+                    if (emitter::isValidSimm12(genTotalFrameSize()))
+                    {
+                        GetEmitter()->emitIns_R_R_I(INS_ld_d, size, REG_SCRATCH, REG_SPBASE, genTotalFrameSize());
```

Comment:
Earlier, we would use `emit_R_R_I` for following cases and now that is not needed?

```
        case INS_andi:
        case INS_ori:
        case INS_xori:
            immFits = (0 <= imm) && (imm <= 0xfff);
            break;
```

Reply:
Delete `emitIns_R_R_Imm` because only here using it and only the ld_d instruction while the other instructions, liking `andi, ori, xori` are not used anywhere.

Reply:
Hi, @kunalspathak 
what should I do next for this PR ?